go.mod

```diff
@@ -5,4 +5,5 @@ go 1.16
 require (
 	github.com/bitrise-io/go-steputils v1.0.2
 	github.com/bitrise-io/go-utils v1.0.2
+	github.com/bitrise-steplib/steps-cache-push v0.0.0-20220520150345-f1aa51a651f8
```

Comment:
Unfortunately we do not use the `v` prefix for step versions and Go does not like a version specification without the `v` prefix. So it locks the dependency to a commit hash.